notepadqq-1.4.8-appdata.patch

```diff
@@ -0,0 +1,36 @@
+diff -Naur notepadqq-1.4.8.orig/support_files/notepadqq.appdata.xml notepadqq-1.4.8/support_files/notepadqq.appdata.xml
+--- notepadqq-1.4.8.orig/support_files/notepadqq.appdata.xml	2018-05-11 00:26:47.000000000 +0200
++++ notepadqq-1.4.8/support_files/notepadqq.appdata.xml	2018-11-27 23:38:43.141200004 +0100
+@@ -1,10 +1,9 @@
+- <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this submitted upstream?

Reply:
> Is this submitted upstream?

Not entirely. The icon tag has already been removed in the git tree, but the release history has not been added yet.
https://github.com/notepadqq/notepadqq/commit/b8394792a002debd85574aeb79800d6ba46a6b20

Reply:
In progress: https://github.com/notepadqq/notepadqq/pull/808